bindings/python/tests/bindings/test_decoders.py

```diff
@@ -72,6 +83,30 @@ def test_decoding(self):
         assert decoder.decode(["<0xE5>", "<0x8f>", "<0xab>", "a"]) == "叫a"
 
 
+class TestFuse:
+    def test_instantiate(self):
+        assert Fuse() is not None
+        assert isinstance(Fuse(), Decoder)
+        assert isinstance(Fuse(), Fuse)
+        assert isinstance(pickle.loads(pickle.dumps(Fuse())), Fuse)
+
+    def test_decoding(self):
+        decoder = Fuse()
+        assert decoder.decode(["My", " na", "me"]) == "My name"
+
+
+class TestStrip:
+    def test_instantiate(self):
+        assert Strip(left=0, right=0) is not None
+        assert isinstance(Strip(left=0, right=0), Decoder)
+        assert isinstance(Strip(left=0, right=0), Strip)
+        # assert isinstance(pickle.loads(pickle.dumps(Strip(left=0, right=0))), Strip)
```

Comment:
Same here, to remove or fix?

Reply:
Hoped it would go unoticed :)
 
 Pickling is a nice touch, but cumbersome.